app/gui2/shared/ast/tree.ts

```diff
@@ -1338,19 +1324,19 @@ export class TextLiteral extends Ast {
     return asOwned(new MutableTextLiteral(module, fields))
   }
 
-  static new(rawText: string, module: MutableModule): Owned<MutableTextLiteral> {
+  static new(rawText: string, module?: MutableModule): Owned<MutableTextLiteral> {
     const escaped = escapeInterpolation(rawText)
     const parsed = parse(`'${escaped}'`, module)
     if (!(parsed instanceof MutableTextLiteral)) {
       console.error(`Failed to escape string for interpolated text`, rawText, escaped, parsed)
-      const safeText = rawText.replaceAll(/[^-+A-Za-z0-9_. ]/, '')
+      const safeText = rawText.replaceAll(/[^-+A-Za-z0-9_. ]/g, '')
       return this.new(safeText, module)
     }
     return parsed
   }
 
   /** Return the value of the string, interpreted except for any interpolated expressions. */
-  contentUninterpolated(): string {
+  get contentUninterpolated(): string {
```

Comment:
But then, I'm confused here. What does it mean to "get text not meant to be interpolated from text meant to be interpolated"?